Packs/MongoDBAtlas/ReleaseNotes/1_0_1.md

```diff
@@ -0,0 +1,6 @@
+
+#### Modeling Rules
+
+##### New: MongoDB Atlas Modeling Rule
+
+<~XSIAM> Created an XDM fields mapping for MongoDB Atlas logs (Available from Cortex XSIAM v2.4).</~XSIAM>
```

Comment:
Since this is only for XSIAM is this tag necessary?

Reply:
Doesn't really do any harm if kept.
If in the future an XSOAR support will be added, it's for the better.